src/wordcount.js

```diff
@@ -227,9 +228,14 @@ export default class WordCount extends Plugin {
 	_calculateWordsAndCharacters() {
 		const txt = modelElementToPlainText( this.editor.model.document.getRoot() );
 
-		this.characters = txt.length;
+		this.characters = txt.replace( /\n/g, '' ).length;
 
-		this.words = ( txt.match( /[_a-zA-Z0-9À-ž]+/gu ) || [] ).length;
+		const wordsMatch = featureDetection.isUnicodePropertySupported ?
+			txt.match( new RegExp( '[\\p{L}\\p{N}\\p{M}\\p{Pd}\\p{Pc}]+', 'gu' ) ) :
+			/* istanbul ignore next */
```

Comment:
Nope :D

You can try and test it: https://github.com/ckeditor/ckeditor5-mention/blob/e8d7d1bc018d3ceeee347dfe7c400c3c2c154c5c/tests/mentionui.js#L448.

Then add also tests for such characters and skip if not supported (we get coverage from Chrome) - https://github.com/ckeditor/ckeditor5-mention/blob/e8d7d1bc018d3ceeee347dfe7c400c3c2c154c5c/tests/mentionui.js#L837

Reply:
Oh, I see that you've done the second step already for skipping tests.